docs/pages/kubernetes-access/getting-started/local.mdx

```diff
@@ -0,0 +1,349 @@
+---
+title: Try Teleport on a Local Kubernetes Cluster
+description: Use this local demo to get started with Teleport on Kubernetes in 10 minutes.
+---
+
+In this guide, we will show you how to set up Teleport on a local Kubernetes
+cluster. You will see how Teleport enables users to access private resources in
+your cluster—all from a single ingress port—so you can manage authentication,
+authorization, and audit.
+
+## The demo environment
+
+Our demo will run using minikube, which deploys Kubernetes on your local
+machine. The cluster will run Kubernetes Dashboard, the official browser UI for
+Kubernetes.
+
+While it is possible to expose the dashboard outside the cluster using the
+`kubectl proxy` command, in our demo the dashboard will only be accessible via
+Teleport.
+
+The **Teleport Auth Service** is the certificate authority for your cluster. It
+issues certificates and conducts authentication challenges. The **Teleport Proxy
+Service** is the cluster frontend, and handles user requests, forwards user
+credentials to the Auth Service, and communicates with other Teleport
+instances—called **Nodes**—that enable access to your infrastructure.
+
+In our demo, one Node is a pod that runs Teleport's Application Service to
```

Comment:
What's the "Application Service" referenced here? Can we be more specific?

Also, since the next sentence talks about "a second pod..." I think we should just say something along the lines of:

```suggestion
We will be running two pods for this demo. The first pod runs...`
```

Reply:
I wrote this back when I was mistakenly using "Node" to refer to individual resource services. I've edited the wording.